handlers/handlers.go

```diff
@@ -232,14 +218,36 @@ func AuthenticateOrRedirectToLogin(
 			http.Redirect(
 				responseWriter,
 				request,
-				paths.LoginOrSignupPath,
+				paths.LoginOrSignup,
 				http.StatusTemporaryRedirect)
 		} else {
 			authenticatedHandlerFunc(responseWriter, request, userId)
 		}
 	}
 }
 
+// GetRedirectHandler is a function which given a path returns a handler that
+// on get requests redirects to the given path.
+func GetRedirectHandler(
+	newPath string,
```

Comment:
`newPath` is weird in the context of this function, because there's no original path to differentiate against

Reply:
`finalPath`?

Reply:
modifying it with `final` is still confusing when there's no non-final path in context. Are you opposed to simply naming it `path`?